op-node/rollup/sequencing/sequencer.go

```diff
@@ -753,6 +754,7 @@ func (d *Sequencer) ConductorEnabled(ctx context.Context) bool {
 }
 
 func (d *Sequencer) SetRecoverMode(mode bool) {
```

Comment:
Now that we have a `SetRecoverMode` setter on the l1os as well, and use it here, we could actually remove the bool from the `NewL1OriginSelector` constructor, since we only really set it while it's already built. It seems redundant. Then we need to call it in the sequencer constructor though. Note that even in the l1os test, you construct it with `false` and only later enable it. So the default l1os is always constructed with this featured turned off and it has to be explicitly enabled with this setter. I think that API makes sense given that it's an optional mode that should only ever be enabled temporarily.

Reply:
I removed the param from the constructor. The setter is invoked here:

https://github.com/ethereum-optimism/optimism/blob/1e220b2604a88e70692a65e389689901cf0138b5/op-node/rollup/driver/state.go#L78-L84